ReSwiftTests/TestFakes.swift

```diff
@@ -31,6 +31,22 @@ extension TestStringAppState: Equatable {
     }
 }
 
+struct TestNonEquatable: StateType {
+    var testValue: NonEquatable
+
+    init() {
+        testValue = NonEquatable()
+    }
+}
+
+struct NonEquatable: StateType {
```

Comment:
this shouldn't need to be `StateType` since its used as a substate. `StateType` is only necessary for top-level state